src/components/ResponsiveContent/ResponsiveContent.styl

```diff
@@ -0,0 +1,7 @@
+@require 'settings/breakpoints'
+
+.ResponsiveContentSize_default
+    padding 1.5rem 2rem
+
+    +small-screen()
+        padding 1rem
```

Comment:
I don't like the ResponsiveContent name, I would prefer Padded or something like this.

In some context, we could have something like:

```
class MyPage extends Component {
  render () {
    return <div>
      <ResponsiveContent>
           Blalbla with margins
      </ResponsiveContent>
      Blabla without margins, for example a table, that could be responsive.
      <ResponsiveContent>
           Re-blabla with margins
      </ResponsiveContent>
    </div> 
  }
```

I think ResponsiveContent is not really the right name since in this context, the content in the middle would certainly be responsive also, but just not padded.